acceptance/single_key_test.go

```diff
@@ -124,12 +124,22 @@ func TestSingleKey(t *testing.T) {
 
 	// Verify that none of the workers encountered an error.
 	var results []result
-	for i := 0; i < *numNodes; i++ {
-		r := <-resultCh
-		if r.err != nil {
-			t.Fatalf("%d: %v", i, r.err)
+	for len(results) != *numNodes {
+		delay := 1 * time.Second
+		if len(results) > 0 {
+			delay = 1 * time.Hour
+		}
+		select {
+		case <-stopper:
+			t.Fatalf("interrupted")
+		case r := <-resultCh:
+			if r.err != nil {
+				t.Fatal(r.err)
+			}
+			results = append(results, r)
+		case <-time.After(delay):
+			log.Infof("%d", atomic.LoadInt64(&expected))
```

Comment:
No, this is just printing out stats every second so that you know the test is proceeding. I'll add a comment to make that clearer.